### ISSUE TITLE
Annotation Editor - Ability to download annotations

### ISSUE THREAD

**Issue by dcravenus** [BYU-ODH/hummedia-client#17]
The annotation editor should allow users to download annotations.

**Comment by webnard** [comment]
I think you can do this if you're a superuser. It's what we use to get
annotations for the IC.

**Comment by dcravenus** [comment]
Good point.  Annotations can be downloaded already (link pictured below).  Harold would like us to also have a link in the annotation editor.
![image](https://cloud.githubusercontent.com/assets/4764559/10032111/57d295f0-613e-11e5-98dc-17a491cd9e35.png)

**Comment by dcravenus** [comment]
Or is there already a link in the editor?

**Comment by webnard** [comment]
No, there's no link in the editor right now. Worth noting that annotations must be saved before the download link will work properly, unless you want to serialize the annotations client side.
